packages/upload/src/vaadin-upload-file.js

```diff
@@ -71,6 +71,11 @@ class UploadFile extends FocusMixin(ThemableMixin(PolymerElement)) {
           border: none;
           box-shadow: none;
         }
+
+        :host([complete]) [part='progress'],
+        :host([error]) [part='progress'] {
+          display: none;
```

Comment:
Is it missing `!important`? 

https://github.com/vaadin/web-components/pull/4870/files#diff-4ec7d253efa6800a2415b532ecb6fb9677b912a37f9443ec3b58f3073c066ff8R79

Reply:
Makes sense, added `!important`.